web3.md

```diff
@@ -293,5 +293,49 @@ module WEB3
     rule <k> #evm_increaseTime => #sendResponse( "result" : Int2String(TS +Int DATA ) ) ... </k>
          <params> [ DATA:Int, .JSONList ] </params>
          <timestamp> ( TS:Int => ( TS +Int DATA ) ) </timestamp>
+```
+
+# eth_estimateGas
+
+```k
+    syntax KItem ::= "#eth_estimateGas"
+ // -----------------------------------
+    rule <k> #halt ~> #eth_estimateGas => #sendResponse( "result": #unparseQuantity( GASUSED ) ) ... </k>
```

Comment:
reasonably sure this is not going to work. gasUsed is a field in the block header that is not going to be populated by executing a transaction.